package.json

```diff
@@ -15,6 +15,7 @@
     "react": "^17.0.1",
     "react-beautiful-dnd": "^13.0.0",
     "react-dom": "^17.0.1",
+    "react-h5-audio-player": "^3.7.0",
```

Comment:
Since you've added a new dependency, I would expect the package-lock file will need to need change. You should probably raise this with the team and see what actions should be taken